supportedBackends/google/pipelines/v2alpha1/src/test/resources/reference.conf

```diff
@@ -0,0 +1,2 @@
+demo.dos.docker-image = "somerepo/dos-downloader:tagged"
```

Comment:
So this will be the docker image link containing the environment for running the Scala script?

Reply:
Nevermind. I found answer to both my questions in the link mentioned in comment! Should have taken a look at it first!